src/csvcubed/models/csvcubedexception.py

```diff
@@ -94,6 +98,10 @@ class CsvcubedExceptionUrls(Enum):
         "http://purl.org/csv-cubed/err/column-definition-not-supported"
     )
 
+    PrimaryKeyColumnTitleCannotBeNone = (
+        "http://purl.org/csv-cubed/err/invalid_title_for_code_list_col_with_primary_key"
```

Comment:
Can you make this shorter and use dashes to be consistent with the others?

Perhaps `http://purl.org/csv-cubed/err/invalid-pk-col-title`?